web-server/src/components/PRTable/PullRequestsTableHead.tsx

```diff
@@ -7,7 +7,7 @@ import {
   TableSortLabel
 } from '@mui/material';
 import { FC } from 'react';
-
+import { saveAs } from 'file-saver';
```

Comment:
I don't think this functionality needs an npm package. You can do this in 2-4 lines of code.